docs/config.md

```diff
@@ -227,6 +227,27 @@ cpu_only_composing_models: <comma-delimited-string-list>
 # Enables the searching of request rate (instead of concurrency)
 [ request_rate_search_enable: <bool> | default: false]
 
+# Minimum percentage of the search space to profile when using Optuna
+[ min_percentage_of_search_space: <int> | default: 5]
+
+# Maximum percentage of the search space to profile when using Optuna
+[ max_percentage_of_search_space: <int> | default: 10]
```

Comment:
5-10% seems like a very small band. Do your experiments show that <5% is generally bad and >10% is overkill?